conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -11,11 +11,9 @@ environment:
   {%- endfor %}
 
   matrix:
-  {%- for case in matrix %}
-    {%- for dep_name, version in case %}
-    {% if loop.first %}- {% else %}  {% endif %}{{ dep_name }}: {{version }}
-    {%- endfor %}
-{% endfor %}
+  {%- for config in configs %}
+    - CONFIG: {{ config}}
+  {% endfor %}
```

Comment:
nit: Could we please dedent the jinja on the bottom? This was done to avoid extra whitespace.

Reply:
please go ahead and make any nitpicks you want as commits to this PR.

Reply:
Sure. Fixed.